extensions/email/src/main/java/org/mule/extension/email/internal/retriever/RetrieverOperations.java

```diff
@@ -44,10 +47,11 @@
      * @return a {@link List} of {@link MuleMessage} carrying all the emails and it's corresponding attributes.
      */
     //TODO: ADD PAGINATION SUPPORT WHEN AVAILABLE
+    @Summary("List all the emails in the given Mailbox Folder")
     public List<MuleMessage> list(@UseConfig RetrieverConfiguration config,
                                   @Connection RetrieverConnection connection,
-                                  @Optional(defaultValue = INBOX_FOLDER) String mailboxFolder,
-                                  @Optional EmailPredicateBuilder matcher)
+                                  @Optional(defaultValue = INBOX_FOLDER) @Summary("Mailbox folder where the emails are going to be fetched") String mailboxFolder,
+                                  @Optional @Summary("Email Matcher which gives the capability of filter the retrieved emails") @DisplayName("Email Matcher") EmailPredicateBuilder matcher)
```

Comment:
I think that Email Matcher is redundant. Matcher is descriptive enough. What else are you going to be matching?